templates/juniper_junos_show_lldp_neighbors.textfsm

```diff
@@ -0,0 +1,15 @@
+Value Required LOCAL_INTERFACE (\S+)
+Value Required PARENT_INTERFACE (\S+)
+Value Required CHASSIS_ID (\S+)
+Value Required PORT_INFO (\S*)
+Value Required SYSTEM_NAME (\S+)
+
+Start
+  ^Local\s+Interface -> LLDP
+
+LLDP
+  ^${LOCAL_INTERFACE}\s*${PARENT_INTERFACE}\s+${CHASSIS_ID}\s+${PORT_INFO}\s+${SYSTEM_NAME} -> Record
+  ^${LOCAL_INTERFACE}\s*
+  ^${PARENT_INTERFACE}\s+${CHASSIS_ID}\s+${PORT_INFO}\s+${SYSTEM_NAME} -> Record
```

Comment:
Do you have an example of the local interface on a line and then the rest of it on a separate line? If so, do you mind adding it to the `.raw` file.

Reply:
Per our discussion, I did not have an example of local interface on one line and the rest on another. So I removed lines 12 and 13.